app/controllers/articles_controller.rb

```diff
@@ -1,11 +1,17 @@
 class ArticlesController < ApplicationController
+  
+  before_action :set_article, only: [:show, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update, :destroy]
+  before_action :authenticate_user!, except: [:index, :show]
+  before_action :publisher_user, only: [:new, :create]
 
   def new
-    @article = Article.new
+    @article = current_user.articles.build
+    @titulo = "Nuevo Articulo"
```

Comment:
it feels like this logic belongs to the view. If you're planning to tweak a shared view title try something like `@article.new_record? ? "Nuevo Articulo" : "Editar Articulo"`